src/conf.h

```diff
@@ -39,6 +39,7 @@ typedef struct {
  */
 struct config_s {
         sblist *basicauth_list;
+        char *basicauthrealm;
```

Comment:
for consistency, "basicauth_realm" would look better.

Reply:
Done... I also squashed all my additionla commits and forced pushed again